cypress/e2e/zenscrape/register.cy.js

```diff
@@ -28,135 +28,135 @@ describe('Zenscrape Register Page  ', () => {
+    it('11. Test to verify the "Features" button of the free plan and should open the popup for free plan', () => {
         cy.get(':nth-child(1) > :nth-child(2) > .btn').contains('Features').click({ force: true });
         cy.get('.modal-header').contains('Free');
         cy.get('.plan-feature-list > li').should('be.visible', 'contain.text', '1.000 requests per month');
         cy.get('.modal-footer > .btn').click({ force: true });
         cy.wait(500);
     });
 
-    it('12. Test to Verify the "Features" button of Small Monthly plan and should open the popup for Small Monthly plan', () => {
+    it('12. Test to verify the "Features" button of small monthly plan and should open the popup for small monthly plan', () => {
         cy.get(':nth-child(2) > :nth-child(2) > .btn').contains('Features').click({ force: true })
         cy.get('.modal-header').contains('Small');
         cy.get('.plan-feature-list > li').should('be.visible', 'contain.text', '250.000 requests per month');
         cy.get('.modal-footer > .btn').click({ force: true });
         cy.wait(500);
     });
 
-    it('13. Test to Verify the "Features" button of Medium Monthly plan and should open the popup for Medium Monthly plan', () => {
+    it('13. Test to verify the "Features" button of medium monthly plan and should open the popup for medium monthly plan', () => {
         cy.get(':nth-child(3) > :nth-child(2) > .btn').contains('Features').click({ force: true });
         cy.get('.modal-header').contains('Medium');
         cy.get('.plan-feature-list > li').should('be.visible', 'contain.text', '1.000.000 requests per month');
         cy.get('.modal-footer > .btn').click({ force: true });
         cy.wait(500);
     });
 
-    it('14. Test to Verify the "Features" button of Large Monthly plan and should open the popup for Large Monthly plan', () => {
+    it('14. Test to verify the "Features" button of large monthly plan and should open the popup for large monthly plan', () => {
         cy.get(':nth-child(4) > :nth-child(2) > .btn').contains('Features').click({ force: true });
         cy.get('.modal-header').contains('Large');
         cy.get('.plan-feature-list > li').should('be.visible', 'contain.text', '3.000.000 requests per month');
         cy.get('.modal-footer > .btn').click({ force: true });
         cy.wait(500);
     });
 
-    it('15. Test to Verify the "Features" button of Professional Monthly plan and should open the popup for Professional Monthly plan', () => {
+    it('15. Test to verify the "Features" button of professional monthly plan and should open the popup for professional monthly plan', () => {
         cy.get(':nth-child(5) > :nth-child(2) > .btn').contains('Features').click({ force: true });
         cy.get('.modal-header').contains('Professional');
         cy.get('.plan-feature-list > li').should('be.visible', 'contain.text', '5 000 000 requests per month');
         cy.wait(500);
```

Comment:
why wait

Reply:
need wait in order to get the modal close button.